tests/contrib/scrapy/test_pipeline_cerberus.py

```diff
@@ -0,0 +1,148 @@
+from __future__ import absolute_import
+import sys
+import os
+import pytest
+from scrapy import Item
+from scrapy.utils.test import get_crawler
+from spidermon.contrib.scrapy.pipelines import ItemValidationPipeline
+from tests.fixtures.items import TestItem, TreeItem
+from tests.fixtures.validators import (
+    cerberus_tree_schema,
+    cerberus_test_schema,
+    test_cerberus_schema_string,
+)
+
+STATS_ITEM_ERRORS = "spidermon/validation/items/errors"
+STATS_TYPES = "spidermon/validation/validators/{}/{}"
+SETTING_CERBERUS = "SPIDERMON_VALIDATION_CERBERUS"
+
+
+def assert_in_cerberus_stats(obj):
+    return "{}".format(STATS_TYPES.format(obj.__name__.lower(), "cerberus"))
+
+
+def the_stats(pipe):
+    return pipe.stats.stats.get_stats(pipe)
```

Comment:
From what I remember, the call in the tests was only `pipe.stats.stats.get_stats()` without `pipe` argument.
If that is accurate, why the change?

Reply:
My bad, I actually was using the `ctrl+d` in VScode to change all duplicate entries to `the_stats(pipe)` which might have to lead it to add one here as well. Fixed. How about now.